app/Http/Requests/FilesUploadRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+use Illuminate\Contracts\Validation\Validator;
+
+class FilesUploadRequest extends Request
+{
+
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        $rules = [
+            'upload' => 'image|max:10240',
+        ];
+
+        return $rules;
+    }
+
+    public function formatErrors(Validator $validator)
+    {
+        foreach ($validator->errors()->all() as $error) {
+            echo '<script type="text/javascript">window.parent.CKEDITOR.tools.callFunction("'.$this->CKEditorFuncNum.'", "", "'.$error.'")</script>';
```

Comment:
Egor, this is just for example
`public function response(Array $errors)
    {
        return Response::json($errors);
    }


    public function formatErrors(Validator $validator)
    {
        return messageToAlertCorner('delete', trans('notification.corner.ad-privilege-error'),
            config('notification.color.error'));
    }`